src/object-registry.js

```diff
@@ -37,6 +45,10 @@ export default class ObjectRegistry {
     this.gc = {};
   }
 
+  /**
+   * Load the objects from an existing ObjectRegistry and store them in this.objects
```

Comment:
For method- and class-level documentation, I'd personally use third-person (which has also been the style in our repo so far :smiley:).